### ISSUE TITLE
Ignore not-yet-decided IPLab issues

### ISSUE THREAD

**Issue by dhendriks** [eclipse-dash/dash-licenses#419]
I get the following diff on Dash license check output in the Eclipse ESCET project:

```diff
-maven/mavencentral/org.hamcrest/hamcrest/3.0, BSD-3-Clause, approved, clearlydefined
+maven/mavencentral/org.hamcrest/hamcrest/3.0, None, restricted, #17661
```

Previously, the dependency (that we get via Eclipse Platform or so), was OK via ClearlyDefined. Now, it is no longer OK. Now, there is an IPLab issue. I don't know who created it, or why, but I didn't. The new IPLab issue is currently under evaluation, so there is no conclusion. This is now interpreted by the Dash license check tool as being restricted.

I think such not-yet-decided IPLab issues should be ignored. They don't forbid the use of the dependency, and ClearlyDefined should then be queried. That is, a not-yet-decided IPLab issue should be treated like a non-existing IPLab issue, I think.

With this change, the automated dependency check wouldn't fail in these kind of situations.

**Comment by waynebeaton** [comment]
I've approved the issue on behalf of the IP Team so that you're not blocked. The line will still change because the authority of the approval has changed; this is by design and I don't intend to change it.

The first thing that I need to sort out is why the new issue was created in the first place. If you got an `approved` response from ClearlyDefined, then so too should other projects. With an `approved` response, there should be no need to open an IPLab issue. AFAICT, the ClearlyDefined record was updated on October 30 2024, so maybe something changed in that data. I'll dig into that.

In parallel, I'll investigate what it would take to introduce a third `undetermined` state.

**Comment by dhendriks** [comment]
> I've approved the issue on behalf of the IP Team so that you're not blocked. The line will still change because the authority of the approval has changed; this is by design and I don't intend to change it.

There were actually 20 new IPLab issues created for dependencies previously approved by ClearlyDefined, see https://gitlab.eclipse.org/eclipse/escet/escet/-/issues/1088#note_2934129. 18/20 got auto-approved. Only 2/20 were restricted dependencies, of which one still remains: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/17667.

**Comment by waynebeaton** [comment]
> Only 2/20 were restricted dependencies, of which one still remains

The IP Team has processed that one as well. So, AFAIK, you are not blocked at this time.

**Comment by dhendriks** [comment]
> The IP Team has processed that one as well. So, AFAIK, you are not blocked at this time.

Indeed. Thanks!
